### PR TITLE
Deploy private deck in default namespace, btw change them to clusterI…

### PR DESCRIPTION
**Pull request by chaodaiG**
…P instead of nodeIP

### CODE REVIEW

File: prow/oss/cluster/deck_private_deployment.yaml [hunk 1]
```diff
@@ -2,6 +2,7 @@ apiVersion: apps/v1
 kind: Deployment
 metadata:
   name: deck-private
+  namespace: default
```
Comment (cjwagner) [inline prow/oss/cluster/deck_private_deployment.yaml:5]: I'm not sure why this defaulted to `test-pods`, but it should be explicit anyways.

File: prow/oss/cluster/deck_private_deployment.yaml [hunk 2]
```diff
@@ -2,6 +2,7 @@ apiVersion: apps/v1
 kind: Deployment
```
Comment (cjwagner) [inline prow/oss/cluster/deck_private_deployment.yaml:2]: This file still has some resources being applied to `test-pods`.

File: prow/oss/cluster/cluster.yaml [hunk 3]
```diff
@@ -94,7 +85,7 @@ metadata:
     kubernetes.io/ingress.global-static-ip-name: "oss-prow"
     kubernetes.io/ingress.class: "gce"
     kubernetes.io/tls-acme: "true"
-    networking.gke.io/managed-certificates: oss-prow-knative-dev,oss-prow-private-knative-dev
+    networking.gke.io/managed-certificates: oss-prow-knative-dev
 spec:
   rules:
   - host: oss-prow.knative.dev
```
Comment (cjwagner) [inline prow/oss/cluster/cluster.yaml:91]: We should probably also temporarily remove the oss-prow-private.knative.dev host rule.

**Review by cjwagner — CHANGES_REQUESTED** [review]

File: prow/oss/cluster/deck_private_deployment.yaml (hunk 2 above)
Comment (chaodaiG) [reply prow/oss/cluster/deck_private_deployment.yaml:2]: There are 2 of them, for binding deck service account with test-pods RBAC roles, not quite sure why deck need them

File: prow/oss/cluster/cluster.yaml (hunk 3 above)
Comment (chaodaiG) [reply prow/oss/cluster/cluster.yaml:91]: removed

File: prow/oss/cluster/deck_private_deployment.yaml (hunk 2 above)
Comment (cjwagner) [reply prow/oss/cluster/deck_private_deployment.yaml:2]: I see, those are for fetching pod logs. I'm not sure if they are actually needed due to the way we use kubeconfig files, but shouldn't hurt.

**Review by cjwagner — APPROVED** [review]

**Pull request merged**